app/build.gradle

```diff
@@ -123,17 +123,15 @@ custom.eachFile() { file ->
     return
   }
   map.put(fileName, file.getAbsolutePath())
-
-
 }
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 25
```

Comment:
No problem, I miss this information during our discussion.

I'm agree with you, it a good idea to keep up to date with our dependencies and SDK.
However, is there any compatibility problems that could occurs by switching to sdk 25 ?
Or it is the platform who define entirely where the application could run ?